tools/tool-base/src/test/java/io/spine/code/fs/js/FileReferenceTest.java

```diff
@@ -63,4 +64,13 @@ void directorySkipRelative() {
         assertEquals("foo/bar", fileReference.directory()
                                              .value());
     }
+
+    @Test
+    @DisplayName("obtain the empty directory path")
+    void emptyDirectory() {
+        FileReference file = FileReference.of("./neighbour.txt");
```

Comment:
Honestly, I would expect that the `/` symbol here would be a system-specific path delimiter. I don't know, but it seems to me that some Windows versions would gladly interpret `.\neighbour.txt` as a file in the current folder.

Reply:
As this is an *imported* file reference, we only use this `/` type of slashes.